src/core/audio/AudioJack.cpp

```diff
@@ -411,6 +436,22 @@ int AudioJack::processCallback( jack_nframes_t _nframes, void * _udata )
 		}
 	}
 
+	if (! m_stopped) {
+		m_inBuffer.reset (new sampleFrame[_nframes]);
+
+		const float gain = mixer()->masterGain();
+		for( int c = 0; c < channels(); ++c ) {
+			jack_default_audio_sample_t *channel_buffer = m_tempInBufs[c];
+
+			for( jack_nframes_t frame = 0; frame < _nframes; ++frame )
+			{
+				m_inBuffer[frame][c] = channel_buffer[frame] / gain;
+			}
+		}
+
+		mixer()->pushInputFrames (m_inBuffer.get (), _nframes);
+	}
+  
```

Comment:
Trailing whitespace here.

Reply:
I couldn't find any whitespace there, are you talking about the newline?

Reply:
I can find two spaces in Line 454, at least in your repository.

Reply:
454 has two spaces after it.  I can't see it with the naked eye myself.  Not sure how @PhysSong spotted it.  Ninja perhaps? :D

Reply:
Yes, I see it. Not sure how I missed it, I searched for spaces. Maybe it was github's merge that inserted it.

Reply:
I found that in the diff page of the merge commit.